server/api/user_handler.py

```diff
@@ -7,7 +7,7 @@
 user_handler = Blueprint('user_handler', __name__)
 
 
-@user_handler.route('/api/v1/users/<user_id>/profile', methods=['GET'])
+@user_handler.route('/api/v1/user/<user_id>/profile', methods=['GET'])
```

Comment:
I think we should keep this as '/api/v1/users/<user_id>/profile'

Reply:
I was looking to make it consistent with the project routes, but I do think plural is better. I'll change them back.